### PR TITLE
Changes following the C++-anonymous meeting.

### PR DESCRIPTION
**Pull request by jhidding**
Changes following the C++-anonymous meeting.

### CODE REVIEW

File: languages/ccpp.md [hunk 1]
```diff
@@ -22,7 +10,27 @@ The latest ratified standard of C++ is C++14. The first standardised version of
...
+* `-I*path*` add path to search for include files
+* `-o*file*` output file
+* `-c` only compile, do not link
+* `-Wall` be more verbose with warnings
+
+And linker flags:
+
+* `-l*library*` links to a library
+* `-L*path*` add path to search for libraries
+* `-shared` make a shared library
```
Comment (LourensVeen) [inline languages/ccpp.md]: Should that have -fPIC as well? That's a compiler option though, but without position independent code shared objects don't work.

File: languages/ccpp.md [hunk 2]
```diff
@@ -37,9 +45,16 @@ With either one of these two (CMake or Autotools), any moderately experienced us
 
 There are many other systems, to name a few: Boost::Build, Perforce JAM, IcMake, Boil, etcetera. A library like Qt even forces its own build system on you. These are not recommended if you don't also supply an option for building with CMake or Autotools.
```
Comment (LourensVeen) [inline languages/ccpp.md]: I'd like a more clear recommendation in this section. Boost::Build sucks, and there seemed to be some enthusiasm about meson in the meeting.

Comment (nielsdrost) [inline languages/ccpp.md]: > Boost::Build ~~sucks~~ does not work for me.

Stay positive people :-) Also, opinion vs facts.

Comment (LourensVeen) [inline languages/ccpp.md]: All right, all right ;-). You're right of course.

Here's a [link to the page about Boost Build on the Boost Library Incubator](http://blincubator.com/tools_bjam/), a website where aspiring Boost contributors can get help getting their library up to the required standards. It lists some pros and cons of Boost Build, and then concludes:

> These disadvantages discouraged me from requiring Boost Build. Any library which is accepted by Boost will have to include support for Bjam or whatever boost requires at the time. But having to sort this out up front was considered too large a burden on library authors.

So in other words, even for libraries that are intended to become part of Boost, it's considered to be too much of a burden to require building with Boost Build from the start.

I've tried to use Boost Build on a project in the past, and never managed to actually get anything to build. I did get CMake to work, but due to missing documentation never really understood what I was doing. I'm now using Meson on another project, and while it's still somewhat in development and has a rough edge or two, at least things build (and fast, too) and I understand what's going on. So I prefer Meson, but that's an opinion again, of course :-).

**Pull request merged**